radis/lbl/base.py

```diff
@@ -2493,6 +2506,9 @@ def Qgas(self, df1, Tgas):
                 return df1["iso"].map(Qgas_dict)
 
         else:  # "iso" not in df:
+            if "gamvdW" in df1.columns:
```

Comment:
```suggestion
            if self.get_conditions()['dbformat'] == 'kurucz'
```
to test more rigorously if the species is an atom